packages/e2e-tests/src/specs/editor/fontCheck/fontCheck.js

```diff
@@ -141,7 +144,7 @@ describe('Font Check', () => {
     await expect(page).toMatchElement('h2', { text: 'Dashboard' });
   });
 
-  it('should redirect to dashboard when pressing ESC', async () => {
+  it('should redirect to dashboard when pressing ESC !', async () => {
```

Comment:
?

Reply:
Leftover from debugging -- watch mode to trigger a change --- will remove.

Reply:
removed.